xc/common/utils/prjxray_create_synth_tiles.py

```diff
@@ -16,11 +16,16 @@ def map_tile_to_vpr_coord(conn, tile):
     c.execute("SELECT pkey FROM phy_tile WHERE name = ?;", (tile, ))
     phy_tile_pkey = c.fetchone()[0]
 
+    c.execute("SELECT pkey FROM tile_type WHERE name = 'NULL'")
+    null_tile_type_pkey, = c.fetchone()
+
     # It is expected that this tile has only one logical location,
     # because why split a tile with no sites?
     c.execute(
-        "SELECT tile_pkey FROM tile_map WHERE phy_tile_pkey = ?",
-        (phy_tile_pkey, )
+        """
+SELECT tile_pkey FROM tile_map INNER JOIN phy_tile ON tile_map.phy_tile_pkey = phy_tile.pkey
+WHERE phy_tile_pkey = ? AND tile_type_pkey != ?
```

Comment:
For clarity in the WHERE statement, please explicitly mention the table where fields come from.  So "tile_pkey" should be "tile_map.tile_pkey", etc.